app/controllers/facility_statements_controller.rb

```diff
@@ -25,15 +25,15 @@ def index
   # GET /facilities/:facility_id/statements/new
   def new_with_search
     @order_details = @order_details.need_statement(@facility)
-    @order_detail_action = :send_statements
+    @order_detail_action = :create_statements
     set_default_start_date if SettingsHelper.feature_on?(:set_statement_search_start_date)
     @layout = "two_column_head"
   end
 
-  # POST /facilities/:facility_id/statements/send_statements
-  def send_statements
+  # POST /facilities/:facility_id/statements/create_statements
+  def create_statements
```

Comment:
I fully support this change in naming, although would simply `create` work as well and make it a little more RESTful?